docs/toc.md

```diff
@@ -19,9 +19,14 @@ Protractor Tests
 Reference
  - [Config File](/lib/config.ts)
  - [Protractor API](/docs/api.md)
+ - [Style Guide](style-guide.md)
+ - [Protractor Syntax vs WebDriverJS Syntax](webdriver-vs-protractor.md)
  - [Browser Support](/docs/browser-support.md)
+ - [Plugins](plugins.md)
  - [Timeouts](/docs/timeouts.md)
  - [The WebDriver Control Flow](/docs/control-flow.md)
+ - [Using TypeScript](typescript.md)
```

Comment:
Maybe this should link to the examples directly, since the TypeScript docs just say "please see our examples"? Unless the plan is to expand that later. Same for the `async`/`await` link...